CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Config option `[cluster]` has been replaced with `[coordinator]`
 * Support for config options `[collectd]` and `[opentsdb]` has been removed; use `[[collectd]]` and `[[opentsdb]]` instead.
 * Config option `data-logging-enabled` within the `[data]` section, has been renamed to `trace-logging-enabled`, and defaults to `false`.
+* The keywords `IF` and `EXISTS` and `NOT` where removed for this release.  This means you no longer need to specify `IF NOT EXISTS` for `DROP DATABASE` of `IF EXISTS` for `CREATE DATABASE`.
```

Comment:
Since this is a list of three things, you should use commas.

> The keywords `IF`, `EXISTS`, and `NOT` were removed in this release.

There's also spelling typos.

> This means you no longer need to specify `IF EXISTS` for `DROP DATABASE` or `IF NOT EXISTS` for `CREATE DATABASE`.